modoboa/core/extensions.py

```diff
@@ -103,17 +103,15 @@ def get_urls(self, category="app"):
         result = []
         for ext_name in list(self.extensions.keys()):
             ext = self.get_extension(ext_name)
-            options = {}
             if category == "api":
                 root = ""
                 pattern = "{}.urls_api"
             else:
                 root = r"^{}/".format(ext.get_url())
-                options.update({"namespace": ext_name})
```

Comment:
Namespace is automatically added by Django now?

Reply:
The namespace mechanism has changed and it now requires that the `app_name` is set in each extension instead - as it is also done in `modoboa.admin` and so. This was reported at first by #1466.

Reply:
Ok.